public/app/features/logs/components/ControlledLogRows.tsx

```diff
@@ -86,9 +86,7 @@ export const ControlledLogRows = forwardRef<HTMLDivElement | null, ControlledLog
         {rest.visualisationType === 'logs' && (
           <LogRowsComponent ref={ref} {...rest} deduplicatedRows={deduplicatedRows} />
         )}
-        {rest.visualisationType === 'table' && rest.panelState && rest.updatePanelState && (
-          <ControlledLogsTable {...rest} />
-        )}
+        {rest.visualisationType === 'table' && rest.updatePanelState && <ControlledLogsTable {...rest} />}
```

Comment:
Not sure if I'm testing this right, so let me know! Locally I go to `http://localhost:3000/explore` and select the table. Then I type a query `{app="faro-shop-frontend"}` and the table does not show. Line 89 also does not even run if a breakpoint is added. Even if I click the logs (I see logs), but come back to table and it's still empty.
![Screenshot 2025-05-06 at 9 13 37 AM](https://github.com/user-attachments/assets/5c9c38f5-75cf-4be9-9c35-dba5868ce39c)

Reply:
Apparently my issues was because I had set `newLogsPanel = true` in `newLogsPanel = false` the fix works 👍